.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1242,14 +1321,20 @@ stages:
             baseImage: $(baseImage)
             command: "BuildLinuxIntegrationTests --framework $(publishTargetFramework) --IncludeTestsRequiringDocker false"
 
+        - task: DownloadPipelineArtifact@2
+          displayName: Download arm64 monitoring home
+          inputs:
+            artifact: linux-monitoring-home-arm64
+            path: $(System.DefaultWorkingDirectory)/shared/bin/monitoring-home
+
         - task: DockerCompose@0
           displayName: docker-compose run --no-deps IntegrationTests
           inputs:
             containerregistrytype: Container Registry
             dockerComposeFileArgs: |
               baseImage=$(baseImage)
               framework=$(publishTargetFramework)
-            dockerComposeCommand: run --no-deps --rm -e baseImage=$(baseImage) -e framework=$(publishTargetFramework) -e IncludeTestsRequiringDocker=false IntegrationTests.ARM64
+            dockerComposeCommand: run --no-deps --rm -e baseImage=$(baseImage) -e framework=$(publishTargetFramework) -e USE_NATIVE_LOADER=true -e IncludeTestsRequiringDocker=false IntegrationTests.ARM64
```

Comment:
Every testing scenario uses the native loader now right (even though they actual deployed artifacts NuGet/dotnet tool etc don't)? So we should be able to remove the `USE_NATIVE_LOADER` variable entirely and do some tidying up? Again, we should do that in a follow up PR if so (not sure if it should be before or after the package layout changes)

Reply:
After doing that change I felt the need to remove it too :) But I wonder if we can do it as a separate PR.

Reply:
yep, definitely